examples/zkapps/09-recursion/src/rollup.ts

```diff
@@ -204,7 +201,7 @@ class RollupContract extends SmartContract {
 
   deploy(args: DeployArgs) {
     super.deploy(args);
-    this.setPermissions({
+    this.account.permissions.set({
```

Comment:
small change, doesn't do anything - we changed what something was called.

Reply:
It was deprecated so,  I just updated to latest.

![Screenshot 2024-02-09 at 12 24 45 AM](https://github.com/o1-labs/docs2/assets/55076843/a96d8202-ff33-4844-9359-c194808d267a)